matrix_client/room.py

```diff
@@ -345,3 +345,13 @@ def add_room_alias(self, room_alias):
             return True
         except MatrixRequestError:
             return False
+
+    def get_joined_members(self):
```

Comment:
This is fine in theory, but it's kinda useless as soon as you want to start operating on each of these members since display names can't identify users.  This function would be more useful to return a userid, or better yet just return a dictionary containing both. 

Hmm, probably need a member class in the future to hold all this information